code/modules/goonhub/api/routes/maps/Maps.dm

```diff
@@ -0,0 +1,8 @@
+
+/// POST /maps/generate
+/// Dispatches a job to process map screenshots and upload them to the web map viewer
+/datum/apiRoute/maps
```

Comment:
```suggestion
/datum/apiRoute/maps/generate
```
mirror.

Reply:
nvm 

![image](https://github.com/goonstation/goonstation/assets/4741640/f3beae06-5abe-4c0c-8bda-cb2bbb5c6a9f)

Reply:
i'll just uhhh not commit that then